package org.joml2.internal.storeload;

import org.joml2.*;
import org.joml2.Math;

public interface DoubleDualQuatRawOps {
    DoubleDualQuat storeUnsafe(DoubleDualQuat self, long address);
    DoubleDualQuat loadUnsafe(long address);
    DoubleDualQuat storeFloatUnsafe(DoubleDualQuat self, long address);
    DoubleDualQuat loadFloatUnsafe(long address);
}
